src/yangbot.py

```diff
@@ -1,186 +1,93 @@
 import traceback
-from src.tools.funcblocker import funcblocker
+from typing import List
+
+import discord
+
+from src.commands_refactor import command_on_message
+from src.auto_on_message_refactor import auto_on_message
+from src.member_join import on_member_join
+from src.member_update import on_member_update
 
 
 class YangBot():
-    def __init__(self, dbconn, client, config, repeated_messages=4):
+    def __init__(self, dbconn, client, config):
         """
-        Initialization of all data and functions of the bot
+        Initialization of all data
         """
-        # Functions
-        self.auto_on_message_list = {}
+        self.debug = False
+        self.client = client
+        self.conn = dbconn
+        self.config = config
+        self.roles = {

```

Comment:
Find a way to get the roles initialized after the bot is ready so we dont' have to edit this everytime a new role is added